src/main/java/seedu/address/model/group/Group.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.group;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a group in TAT.
+ * A group is anything that would include a
+ * group of students such as a lab or tutorial.
+ */
+public class Group {
+
+    /**
+     * Represents a group type.
+     * Can be a lab or a tutorial.
+     */
+    public enum GroupType {
+        LAB,
+        TUTORIAL;
+    }
+
+    private final String identifier;
+    private final GroupType groupType;
+    //TODO: ArrayList will contain Students
+    private final ArrayList students;
+
+
+    /**
+     * Constructs a group object.
+     *
+     * @param identifier identifies the group. For example,
+     *                   the tutorial code for a tutorial, etc.
+     */
+    public Group(String identifier, String type) {
```

Comment:
Consider taking a param of `GroupType` for `type` directly, so that parsing is not the responsibility of the group